spec/unit/controllers/services/service_instances_controller_spec.rb

```diff
@@ -885,6 +885,53 @@ def stub_delete_and_return(status, body)
             expect(last_response.status).to eq(400)
             expect(decoded_response['code']).to eq(60002)
           end
+
+          context 'when a service instance share exists between spaces' do
+            let(:source_space) { Space.make(organization: space.organization) }
+            before do
+              source_space.add_developer(developer)
+
+              service_instance = create_managed_service_instance(accepts_incomplete: 'false', space: source_space)
+              service_instance.add_shared_space(space)
```

Comment:
**Copy from PR 980**: We got a bit confused initially reading through this because of the var name space since there's more than one space being referred to. We thought the var name source_space was great, so perhaps something like target_space would be more descriptive as well.

Reply:
Hey @anniesing,

We replied to this, over on #980 

Thanks

alex and @deniseyu